test/TestAppUtils/VisualTreeDumper.cs

```diff
@@ -142,7 +142,7 @@ private static void WalkThroughTree(DependencyObject node, Visitor visitor)
         public class DefaultFilter : IFilter
         {
             private static readonly string[] _propertyNamePostfixWhiteList = new string[] {"Brush", "Thickness"};
-            private static readonly string[] _propertyNameWhiteList = new string[] {"Background", "Foreground", "Padding", "Margin", "RenderSize", "Visibility", "Name"};
+            private static readonly string[] _propertyNameWhiteList = new string[] {"Background", "Foreground", "Padding", "Margin", "RenderSize", "Visibility", "Name", "CornerRadius"};
```

Comment:
>CornerRadius [](start = 170, length = 12)

+BorderThickness?

Reply:
We have "Thickness" in the propertyNamePostfixWhiteList, that should cover BorderThickness, StrokeThickness, RevealBorderThickness, etc.